src/Compute/Compute/Extension/AEM/AEMHelper.cs

```diff
@@ -90,26 +90,26 @@ internal StorageAccount GetStorageAccountFromCache(string accountName)
         }
         internal string GetResourceGroupFromId(string id)
         {
-            var matcher = new Regex("/subscriptions/([^/]+)/resourceGroups/([^/]+)/providers/(\\w+)");
```

Comment:
@MSSedusch Rather than update this code, is there any reason you can't use our common code implementation found here: https://github.com/Azure/azure-powershell-common/blob/master/src/ResourceManager/Version2016_09_01/Utilities/Models/ResourceIdentifier.cs? If there is any functionality missing from that common code that will cause this to fail, please let me know and we can see about updating it.

Reply:
no reason. I was not aware of it :) I replaced it